files/202405/0001-iccpd-enabled-in-202405.patch

```diff
@@ -0,0 +1,25 @@
+From c092e211582c78efb143c79f9aede98a746d36f0 Mon Sep 17 00:00:00 2001
+From: nipunjoy <njoy@marvell.com>
+Date: Mon, 23 Sep 2024 10:19:47 +0000
+Subject: [PATCH] iccpd enabled in 202405
+
+---
+ rules/config | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/rules/config b/rules/config
```

Comment:
As discussed let us create a Makefile.work patch to community and maintain that patch internally till it gets merged.

Reply:
updated the patch file.